src/auth/containers/emailVerification/EmailVerificationPage.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from "react";
+
+import { ThunkDispatch } from "redux-thunk";
+
+import { AnyAction } from "redux";
+
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  CardActions,
+  Typography,
+  Button,
+  Grid,
+} from "@material-ui/core";
+
+import { connect } from "react-redux";
+
+import { Link } from "react-router-dom";
+
+import * as H from "history";
+
+import { EmailVerification } from "../../core/redux/actions";
+
+import SideImage from "../../components/sideImage/SideImage";
+
+const qs = require("query-string");
```

Comment:
this can be changed to `import`

Reply:
This particular library needs to be imported like this. Given in their npm usage guidelines. It doesn't work right if import is used.